ui/src/designer/widget.js

```diff
@@ -56,7 +56,7 @@ let Widget = function(id, data, regionId = null, layoutObject = null) {
     this.mediaIds = data.mediaIds;
 
     // Widget colouring
-    if(playlistRegionColouring === 'Permissions Colouring') {
```

Comment:
I think we might need to cater for old settings values here - perhaps we can have it in an `or` ?